Added custom metrics to Widget Builder (#10195)
getsentry/sentry-docs: docs/product/dashboards/widget-builder/index.mdx

- Crash rates for your latest release
- Session count across releases

### Custom Metrics

> Choosing "Custom Metrics" allows you to query your custom metrics data in the same way you would in the [Metrics Explorer Page](/product/metrics/metrics-explorer-page/). Some example widgets might include:

Author: @vivianyentran that's actually not quite so 😅 you can still also plot OOTB metrics like transaction.duration and others.

- Tracking your number of user signups
- Tracking API request payloads
- Processing time for a task

> Please note that the widget builder UI is slightly different than for other datasets, to allow for an optimal experience when querying metrics.

Author: ```suggestion
Please note that the widget builder UI is slightly different than for other datasets in order to optimize the experience for querying metrics.
```

> ![Metrics widget builder.](./img/metrics-widget-builder.png)

Author: ```suggestion
![Custom metrics widget builder.](./img/widget-builder-custom-metrics.png)
```
